controllers/limitador_controller.go

```diff
@@ -129,9 +132,58 @@ func (r *LimitadorReconciler) reconcileSpec(ctx context.Context, limitadorObj *l
 	if err := r.reconcilePdb(ctx, limitadorObj); err != nil {
 		return ctrl.Result{}, err
 	}
+
+	if err := r.reconcilePodLimitsHashAnnotation(ctx, limitadorObj); err != nil {
+		return ctrl.Result{}, err
+	}
+
 	return ctrl.Result{}, nil
 }
 
+func (r *LimitadorReconciler) reconcilePodLimitsHashAnnotation(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) error {
+	podList := &v1.PodList{}
+	options := &client.ListOptions{
+		LabelSelector: labels.SelectorFromSet(limitador.Labels(limitadorObj)),
+		Namespace:     limitadorObj.Namespace,
+	}
+	if err := r.Client().List(ctx, podList, options); err != nil {
+		return err
+	}
+
+	if len(podList.Items) == 0 {
+		return errors.New("no limitador pods founds")
```

Comment:
should requeue? not an error if eventually they (pods) should be there

Reply:
Hmm, true, I'll update 👍